### PR TITLE
ava-dev

### PR DESCRIPTION
**Pull request by algodiva**

### CODE REVIEW

File: src/hex/algorithms.rs [hunk 1]
```diff
@@ -211,4 +227,10 @@ mod tests {
             ]
         );
     }
+
+    #[test]
+    fn reflect() {
+        assert_eq!(axial!(1,1).reflect(None, Axes::Q), axial!(1, -2));
+        assert_eq!(axial!(1,1).reflect(Some(axial!(1,2)), Axes::Q), axial!(1, 3));
```
Comment (sevriarch) [inline src/hex/algorithms.rs:234]: Might be useful to have a test case where the value of `q` changes.

File: src/hex/coordinate.rs [hunk 2]
```diff
@@ -97,6 +111,7 @@ mod tests {
         assert!(axial!(4, 2) - axial!(1, 3) == axial!(3, -1));
     }
 
+    #[allow(clippy::erasing_op)]
```
[the remark this answers is not in the document]
Comment (algodiva) [reply src/hex/coordinate.rs:114]: The compiler detects that multiplying an int by 0 erases the data and in another situation it's likely better to just use a constant 0.

see: https://rust-lang.github.io/rust-clippy/master/index.html#erasing_op

File: src/hex/algorithms.rs (hunk 1 above)
Comment (sevriarch) [reply src/hex/algorithms.rs]: testing: fixed in https://github.com/algodiva/Gridly/pull/8/commits/3a8c1621da359af7681d20a6f5ee2a8028b4104f

Comment (sevriarch) [reply src/hex/algorithms.rs]: New test cases look good to me.

**Review by sevriarch — APPROVED** [review]
LGTM

**Pull request merged**